app/controllers/conversations_controller.rb

```diff
@@ -2,7 +2,22 @@ class ConversationsController < ApplicationController
   before_action :signed_in_flash, only: [:create]
   # before_action :participant?, only: [:show]
   def index
-    @conversations = Conversation.all.order("created_at DESC")
+    if params[:order] == "totalpoints"
+      @conversations = Kaminari.paginate_array(Conversation.all.sort{ |b,a| a.sender.average_score + a.recipient.average_score <=> b.sender.average_score + b.recipient.average_score }).page(params[:page]).per(10)
+    elsif params[:order] == "message"
+        @conversations = Kaminari.paginate_array(Conversation.all.sort{ |b,a| a.messages.count <=> b.messages.count }).page(params[:page]).per(10)
```

Comment:
Line is too long. [146/80]<br>Use 2 (not 4) spaces for indentation.<br>Space missing after comma.<br>Space missing to the left of {.